content/docs/studio/get-started.md

```diff
@@ -1,13 +1,15 @@
 # Get Started with Iterative Studio
 
-The following video provides you a quick overview of how to use Iterative
+The following two videos provide you a quick overview of how to use Iterative
 Studio.
 
 > Note that we have renamed DVC Studio to Iterative Studio and Views to
 > Projects.
 
 https://www.youtube.com/watch?v=hKf4twg832g
 
+TODO: Add a video about live metrics and plots tracking
```

Comment:
Not sure it's a good idea to put 2 videos at the top of the Get Started. In fact for DVC we hid all the GS videos; See [Get Started: Data Versioning](https://dvc.org/doc/start/data-management/data-versioning):

<img width="1009" alt="image" src="https://user-images.githubusercontent.com/1477535/199325133-5605cba0-10fd-4a10-b4c6-a53a57f17f39.png">

Reply:
Removed the second video. I don't think we need it here.

Reply:
Sounds good, thanks.